bin/packages/build.js

```diff
@@ -87,39 +97,50 @@ function buildFiles( files ) {
 		return accumulator;
 	}, { jsFiles: new Set(), scssPackagePaths: new Set() } );
 
-	buildPaths.jsFiles.forEach( buildJsFile );
-	buildPaths.scssPackagePaths.forEach( buildPackageScss );
+	return Promise.all( [
+		...buildPaths.jsFiles.map( buildJsFile ),
```

Comment:
I was wrong in thinking that by the previous implementation using `forEach`, these were arrays, when in fact they're of type `Set`. Thus, rebuilds fail with an error:

```
[1] /Users/andrew/Documents/Code/vvv/www/editor/htdocs/wp-content/plugins/gutenberg/bin/packages/build.js:100
[1] 		...buildPaths.jsFiles.map( buildJsFile ),
```

Two alternatives are:

- Don't track them as a `Set`, rather as an array and guarantee uniqueness before pushing to array.
- Convert `Set` to array at this line before the map, i.e. `...[ ...buildPaths.jsFiles ].map( buildJsFile ),`